sunpy/net/tests/test_vso.py

```diff
@@ -438,8 +438,8 @@ def test_vso_hmi(client, tmpdir):
     # Extract the DRIs from the request
     dris = dr.request.datacontainer.datarequestitem
 
-    # 3 HMI series and one AIA
-    assert len(dris) == 4
+    # 9 HMI files and 81 AIA
```

Comment:
Why have these numbers changed?

Reply:
Earlier in `dris` there used to be 3 HMI datarequestitems, and one AIA. But it was like they had array of fileids containing more than one fileid.
For example, the AIA one had 81 files for one item, and hmi had 3 each, so the total files accessible where still 90 but dris list had four items only.